src/javascript/DesignSystem/MultipleInput/MultipleInput.jsx

```diff
@@ -50,6 +48,7 @@ const MultipleInputComponent = ({classes, creatable, readOnly, ...props}) => {
                 styles={classes}
                 isDisabled={readOnly}
                 {...props}
+                isReadOnly={readOnly}
```

Comment:
what's the diff btw isDisabled and isReadOnly?

Reply:
Ask to designer for the answer... It was the major pain point on this pull request :/